evergreen/evergreen.yml

```diff
@@ -14,7 +14,7 @@ command_type: system
 
 # Protect ourself against rogue test case, or curl gone wild, that runs forever
 # 12 minutes is the longest we'll ever run
-exec_timeout_secs: 1800
+exec_timeout_secs: 4800
```

Comment:
I have changed back. The Linux builds were taking too long because of another error that I resolved, in which all target test variants were running on every build.